mesonbuild/msubprojects.py

```diff
@@ -318,9 +317,14 @@ def run(options):
         wraps = [wrap for name, wrap in r.wraps.items() if name in options.subprojects]
     else:
         wraps = r.wraps.values()
+    all_types = ['file', 'git', 'hg', 'svn']
```

Comment:
This should probably be a global or class variable to share the list between here and the help string (earlier comment).

Reply:
Made it global in wrap.py where it belongs.